modules/light-clients/08-wasm/types/update.go

```diff
@@ -13,11 +14,7 @@ var _ exported.ClientState = (*ClientState)(nil)
 
 type (
 	verifyClientMessageInnerPayload struct {
-		ClientMessage clientMessage `json:"client_message"`
-	}
-	clientMessage struct {
-		Header       *Header       `json:"header,omitempty"`
-		Misbehaviour *Misbehaviour `json:"misbehaviour,omitempty"`
+		ClientMessage *ClientMessage `json:"client_message"`
```

Comment:
This is how the payload to the contract call looks after merging `Header` and `Misbehaviour`.